packages/ws-server/src/controllers/index.ts

```diff
@@ -56,7 +56,24 @@ const handleSendingRequestsToRelay = async ({
 
   try {
     const resolvedParams = resolveParams(method, params);
-    const txRes = await relay.eth()[method.split('_')[1]](...resolvedParams, requestIdPrefix);
+    const [service, methodName] = method.split('_');
+    let txRes;
+    switch (service) {
+      case 'debug':
+        txRes = await relay.debug()[methodName](...resolvedParams, requestIdPrefix);
+        break;
+      case 'eth':
+        txRes = await relay.eth()[methodName](...resolvedParams, requestIdPrefix);
+        break;
+      case 'get':
```

Comment:
are we supporting any `get` or `debug` pathways yet?

Reply:
@Nana-EC asked that the pathways be specific, in the above comments.